benchmarks/chained-function-serving/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-FROM vhiveease/golang-builder AS builder
+FROM vhiveease/golang-builder:1.17.13-buster AS builder
```

Comment:
We have now a build-ci to create base images.
Maybe you can add it there?
https://github.com/ease-lab/vSwarm/blob/main/utils/base-images/Dockerfile

Reply:
okay, will add.

Reply:
I am planning to upgrade golang in a new PR. Shall I do it there too?

Reply:
You mean to go 1.18?
I think you should create another PR and then just 
Change [this](https://github.com/ease-lab/vSwarm/blob/9a11983727ee87e580960a6f232d3da8077307ba/utils/base-images/Dockerfile#L25) line to 1.18

Then we can automatically rerun all CI test to see if images that depend on this base image still work

Reply:
Sure. This will reduce merge conflicts. 
I am going ahead with the change.